mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping

```diff
@@ -0,0 +1,31 @@
+CLASS net/minecraft/unmapped/C_lygsomtd net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly
```

Comment:
```suggestion
CLASS net/minecraft/unmapped/C_lygsomtd net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomlyTask
```
Should this have the `Task` suffix to be consistent with other classes in this package?

Also I don't know if there's a convention on where adverbs that modify the verb of task class names go, but it should probably be brought up (`SetEntityLookTargetRandomly` vs `RandomlySetEntityLookTarget`).